circuit_setup/scripts/prepare_mdl_prover.py

```diff
@@ -269,7 +268,7 @@ def ymd_to_daystamp(ymd, is_bytes=False, has_time=False):
     exit(-1)
 
 if config['alg'] == 'ES256':
-    # See https://www.rfc-editor.org/rfc/rfc7515#appendix-A.3.1 for ECDSA encoding details in JWTs, the signature is R||S
+    # See https://www.rfc-editor.org/rfc/rfc7515#appendix-A.3.1 for ECDSA encoding details in JWTs, the signature is R||S FIXME: why are we referencing JWT?
```

Comment:
The (r,s) values of the ECDSA signature are encoded in the same way. It would be clearer to say "See ... for ECDSA encoding details, the signature is R||S"

Reply:
Updated